tool/src/test/java/org/wildfly/security/tool/MaskCommandTest.java

```diff
@@ -63,9 +64,9 @@ public void testMissingSaltAndIteration() {
         String retVal = executeCommandAndCheckStatusAndGetOutput(args);
         String[] retValLines = retVal.split(LINE_SEPARATOR);
 
-        assertTrue("Message about invalid salt parameter must be present", retValLines[0].contains("Invalid \"salt\" parameter. Generated value"));
-        assertTrue("Message about invalid iteration parameter must be present", ("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.").equals(retValLines[1]));
-        assertTrue("Message about invalid salt parameter must be present", retValLines[2].contains("MASK-"));
+        assertEquals("Invalid \"salt\" parameter. Generated value", retValLines[0]);
+        assertEquals("Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.",retValLines[1]);
```

Comment:
for all 3 statements, we can keep the original messages that indicaet why this assertion should pass:
```suggestion
        assertEquals("Message about invalid salt parameter must be present", "Invalid \"iteration\" parameter. Default value \"" + defaultIteration + "\" will be used.",retValLines[1]);
```

Reply:
@ashley-abdelsayed98 I am passing message,expected,actual in the assertEquals method for all the three statements now. This is how it needs to be done. Sorry about the previous commits. Hope so this one is correct. Please let me know if any alteration is required.